migrator/migrations/1_create_users_table.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE public.users
```

Comment:
`CREATE TABLE` use the `public` schema but  `DROP TABLE` does not

Reply:
by default CREATE TABLE uses public schema

Reply:
fixed